vic/drivers/classic/src/vic_classic.c

```diff
@@ -247,14 +235,17 @@ main(int   argc,
                    Calculate cell average values for current time step
                 **************************************************/
                 put_data(&all_vars, &atmos[rec], &soil_con, veg_con, veg_lib,
-                         &lake_con, out_data, &save_data, rec);
+                         &lake_con, out_data[0], &save_data);
```

Comment:
out_data is size `[ngridcells, N_OUTVARS]`. This just passes the 1d array of `N_OUTVARS`.